### ISSUE TITLE
setValue is protected!

### ISSUE THREAD

**Issue by ahmadazimi** [kristijanhusak/laravel-form-builder#136]
Hi there,

Would you please explain why you've made form feild ```setValue``` method proteced?

I wanna set field value dynamiccally through the code, I need to know ```valueProperty``` of the field or call ```setValue``` method.

But both are protected and I can't use neither of them. Is there any way to set field value by its instance? I mean after initializing filed and just before rendering.

```php
$form->add($name, 'select', $options);
// later in the code:
$form->getField($name)->setValue(15);
```

**Comment by kristijanhusak** [comment]
Hi @ahmadazimi ,

There were no need to put it as public, i haven't even noticed that because i never used it that way. It was mostly from model, or in view as an option.

I fixed it in the latest commit on the master branch.

**Comment by ahmadazimi** [comment]
wow :-)

Honestly, I'm working on a form generator based on ```JSON``` file which uses a JSON file for each form to handle all form functionality (fields, validations, access permissions and etc.).

Thank you for your attention and do you know when it will be available on the Composer repository?

**Comment by ahmadazimi** [comment]
BTW: What do you think about adding below methods to ```FormField``` class?

  * ```public function getValueProperty()```
  * ```public function getDefaultValueProperty()```

**Comment by kristijanhusak** [comment]
I added those two now.

Everything is available at `dev-master` branch.

Thanks.

**Comment by kristijanhusak** [comment]
Everything available in latest version (1.6.20). Thanks!
